mlpf/pipeline.py

```diff
@@ -194,7 +204,7 @@ def train(config, weights, ntrain, ntest, nepochs, recreate, prefix, plot_freq,
     model.fit(
         ds_train.repeat(),
         validation_data=ds_test.repeat(),
-        epochs=initial_epoch + config["setup"]["num_epochs"],
+        epochs=config["setup"]["num_epochs"],
```

Comment:
why did you change this? I think it was correct.

Reply:
My thinking is the following. Let's say `config["setup"]["num_epochs"]` is `100` and we resume an interrupted training from epoch `20`. Then `initial_epoch` will be `20` and `initial_epoch + config["setup"]["num_epochs"]` will be 120, right? I think it's more intuitive that `config["setup"]["num_epochs"]` should be the total number of epochs to run before completing the training, rather than the additional number of epochs to run from the resumed point. This is a matter of taste I suppose. What do you think?